pylhc_submitter/job_submitter.py

```diff
@@ -293,6 +293,7 @@ def main(opt):
         opt.executable,
         opt.script_arguments,
         opt.script_extension,
+        (opt.run_local or opt.dryrun)

```

Comment:
I would remove dryrun from there. You want to have this warning on dry-run already

Reply:
Yes and no.
there is a couple of scenarios where one might want to create the >100k job dir and files.
E.g. if this limit does not apply to your user, you can submit after using the `.sub` file.
other edge case might be if you want a granular submission, i.e. specify the jobs and not use the full inner product ..
one could then create the jobs, delete the outputdir for the case one would like and then using `resume_jobs` tun only those .. though this is very hacky..

Reply:
Then I'd not raise a value error at all but only always print the warning, in case the user actually has a larger job-limit